styles.css

```diff
@@ -312,26 +349,161 @@ header .menu-button {
   text-align: center;
   color: #42526e;
 }
+
+@media only screen and (min-width: 1080px) {
+  .hero-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: auto;
+    height: 1100px;
+    background-image: url("images/backround-images/desktop-backgrounds/header-background-desktop.svg");
+    background-size: 100% auto;
+  }
+
+  .hero-section h1,
+  .hero-section h2,
+  .hero-secondary {
+    width: 770px;
+  }
+
+  .hero-section h1,
+  .hero-section h2 {
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .hero-section h1 {
+    margin-top: -250px;
+    padding-top: 0;
+  }
+
+  .hero-secondary {
+    margin-bottom: 24px;
+    line-height: 32px;
+  }
+
+  .projects-header .indicator-icon {
+    display: none;
+  }
+
+  .projects {
+    grid-template: 50% 50% / 1fr 1fr 1fr;
+    padding: 0 106px 0 117px;
+  }
+
+  .project {
+    min-width: 369px;
+    height: auto;
+    margin: 12px;
+  }
+
+  #about-me .about-me-top {
+    padding-top: 152px;
+    align-self: center;
+    background:
+      url("images/backround-images/desktop-backgrounds/about-me-background1-desktop.svg"),
+      url("images/backround-images/desktop-backgrounds/about-me-background2-desktop.svg");
+    background-position: 0 90%, 100% 35%;
+    background-repeat: no-repeat, no-repeat;
+    background-size: 15% auto, 13% auto;
+  }
+
+  .about-me-para {
+    width: 770px;
+    align-self: center;
+  }
+
+  #about-me .technologies {
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 24px;
+  }
+
+  #get-in-contact {
+    flex-flow: row wrap;
+    padding: 0 111px;
+    margin-top: 118px;
+    justify-content: center;
+    background-image:
+      url("images/backround-images/desktop-backgrounds/contact-form-background1-desktop.svg"),
+      url("images/backround-images/desktop-backgrounds/contact-form-background2-desktop.svg"),
+      url("images/backround-images/desktop-backgrounds/contact-form-background3-desktop.svg");
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: 0 80%, 20% 75%, 100% 45%;
+    background-size: 14% auto, 11% auto, 15% auto;
+  }
+
+  #get-in-contact .primary-text {
+    font-size: 30px;
+    max-width: 463px;
+    margin: 24px;
+  }
+
+  #get-in-contact form {
+    width: 684px;
+  }
+
+  #get-in-contact form input,
+  #get-in-contact form textarea {
+    width: 684px;
+  }
+
+  #get-in-contact form button {
+    align-self: baseline;
+    margin-left: 0;
+  }
+
+  #get-in-contact .my-email {
+    display: none;
+  }
+
+  #get-in-contact hr {
+    margin-top: 200px;
+    border: 1px solid #dfe1e6;
+    width: 100%;
+  }
+}
+
+@media only screen and (max-width: 990px) {
+  .header-desktop {
+    display: none;
+  }
+
+  hr {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 990px) {
+  .header-mobile {
+    display: none;
+  }
+}
```

Comment:
Cant this be done at the 1080px media query and you can use the default css mobile first instead of the max-width media query